ebook/1_latex2html.py

```diff
@@ -0,0 +1,737 @@
+# by Torben Menke https://entorb.net
```

Comment:
Add a hash-bang line please:

```
#!/usr/bin/env python3
```

and make executable, even if you keep the instructions to run with "python3".

Reply:
As I am coding on windows, I do not think I can set the executable flag

Reply:
Apparently:

```
git update-index --chmod=+x file
```

Reply:
ah, thanks for teaching me. Should have asked google myself.
fixed now